### PR TITLE
[tools] corrected warnings werf

### PR DESCRIPTION
**Pull request by alexey-gavrilov-flant**
## Why do we need it, and what problem does it solve?
Fixed generation of config chunks for werf.
Warnings are distracting and have some risk for incorrect builds.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: tools
type: fix
summary: fixed generation of config chunks for werf
impact_level: low
```

### CODE REVIEW

File: tools/build_includes/modules-with-dependencies-BE.yaml [hunk 1]
```diff
@@ -1,73 +1,19 @@
 # Code generated by tools/build.go; DO NOT EDIT.
 - add: /ee/be/modules/040-node-manager/openapi/config-values.yaml
   to: /deckhouse/modules/040-node-manager/openapi/config-values.yaml
-  excludePaths:
```
Comment (name212) [inline tools/build_includes/modules-with-dependencies-BE.yaml:4]: Why do we delete here?

**Review by name212 — CHANGES_REQUESTED** [review]
one question

File: tools/build_includes/modules-with-dependencies-BE.yaml (hunk 1 above)
Comment (alexey-gavrilov-flant) [reply tools/build_includes/modules-with-dependencies-BE.yaml:4]: This is pointless for single files, they will end up in the image anyway. 
The excludePaths are added to the path above.

**Comment by alexey-gavrilov-flant** [comment]
FE redaction
image

**Comment by alexey-gavrilov-flant** [comment]
EE redaction
image

**Comment by alexey-gavrilov-flant** [comment]
SE redaction
image

**Comment by alexey-gavrilov-flant** [comment]
BE redaction
image

**Comment by alexey-gavrilov-flant** [comment]
CE redaction
image

**Review by name212 — APPROVED** [review]

**Pull request merged**